tests/unit/autowiring/autowired_decorator_unit_test.py

```diff
@@ -164,3 +164,24 @@ def f(a: AutowiredMockA, b: AutowiredMockB, c: AutowiredMockC):
         assert parameters["a"] is None
         assert parameters["b"] is AutowiredMockB.inject()
         assert parameters["c"] is None
+
+    def test__autowired__injects_named_autowired_args_when_named_args_defined_by_the_caller(self):
```

Comment:
This is subtle variation on the previous test, reversing the `b, c` args showed the issue fixed by this change